VAMobile/src/styles/themes/standardTheme.ts

```diff
@@ -72,6 +72,10 @@ const fontSizes = {
     fontSize: 20,
     lineHeight: 30,
   },
+  CrisisLineButton: {
+    fontSize: 16,
+    lineHeight: 20.11,
```

Comment:
I believe setting this to `20` should be fine, as I don't think the .11 would actually take effect

Reply:
fixed!